clients/include/norm.hpp

```diff
@@ -262,6 +262,8 @@ bool norm_check(double norm_error)
 {
     if(std::is_same<T, float>{})
         return norm_error < 0.00001;
+    if(std::is_same<T, double>{})
+        return norm_error < 0.00001;
```

Comment:
in Tensile client, threadhold is 0.000000000001; // 15 digits of precision - 2

Reply:
Fixed